docs/src/index.md

```diff
@@ -1,14 +1,185 @@
 ```@meta
-CurrentModule = FluorophoreColors
+CurrentModule = MultiChannelColors
 ```
 
-# FluorophoreColors
+# MultiChannelColors
 
-Documentation for [FluorophoreColors](https://github.com/JuliaImages/FluorophoreColors.jl).
+[MultiChannelColors](https://github.com/JuliaImages/MultiChannelColors.jl) aims to support "unconventional colors," such as might arise in applications like multichannel fluorescence microscopy and hyperspectral imaging. Consistent with the philosophy of the [JuliaImages ecosystem](https://juliaimages.org/latest/), this package allows you to bundle together the different color channels into a "color object," and many color objects can be stored in an array. Having each entry of the array represent a complete pixel or voxel makes it much easier to write generic code supporting a wide range of image types.
 
-```@index
+## Installation
+
+Install the package with `add MultiChannelColors` from the `pkg>` prompt, which you access by typing `]` from the `julia>` prompt. See the [Pkg documentation](https://pkgdocs.julialang.org/v1/getting-started/) for more information.
+
+## Usage
+
+Use the package interactively or in code with
+
+```jldoctest demo
```

Comment:
Just a confirmation: is the `jldoctest` deliberately used here rather than the [`@repl` block](https://juliadocs.github.io/Documenter.jl/stable/man/syntax/#@repl-block)?

Reply:
Yes, though there are many ways to solve the same problem. Basically I want doctests to make sure the code keeps working, and I don't want to have to use `setup=:(using MultiChannelColors)` on each doctest. So I created a named doctest, `demo`, and am loading the package in the first instance.